apps/auth/main.py

```
@@ -0,0 +1,51 @@
+from fastapi import FastAPI, HTTPException, Response
+from fastapi import FastAPI
```

Ruff (0.8.2)

Redefinition of unused `FastAPI` from line 1

Remove definition: `FastAPI`

(F811)